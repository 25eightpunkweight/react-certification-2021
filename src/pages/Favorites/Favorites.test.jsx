import React from 'react';
import renderer from 'react-test-renderer';
import TestAppContainer from '../../test/TestAppContainer';
import FavoritesPage from './Favorites.page';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <TestAppContainer>
        <FavoritesPage />
      </TestAppContainer>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
