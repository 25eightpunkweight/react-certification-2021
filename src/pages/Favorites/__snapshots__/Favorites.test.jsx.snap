// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="sc-bdnxRM huhLlJ"
>
  <div
    className="sc-gtsrHT kJGVrV"
  >
    <div
      className="sc-dlnjwi kfZIKC"
    />
  </div>
</section>
`;
